src/linter/linter.ts

```diff
@@ -372,6 +372,33 @@ function checkUnmatchedPatterns(patterns: FilePattern[], patternsMatch: Set<stri
 		return acc;
 	}, [] as FilePattern[]);
 
+	// Check if the Glob pattern is actually a file that exists, but the linter is not able to process.
+	if (unmatchedPatterns.length) {
+		const notProcessedFiles = new Set<string>();
+		const rootFileReader = createReader({
+			fsBasePath: options.rootDir,
+			virBasePath: "/",
```

Comment:
We can exclude node_modules, .git etc by providing an `excludes` parameter here:

```suggestion
			virBasePath: "/",
			excludes: [""]
```